tools/refguide_check.py

```diff
@@ -239,7 +281,26 @@ def get_all_dict(module):
 
 
 def compare(all_dict, others, names, module_name):
-    """Return sets of objects only in __all__, refguide, or completely missing."""
+    """
+    Return sets of objects only in __all__, refguide, or completely missing.
```

Comment:
Return sets of objects from `all_dict`. Will return three sets: {in ``module_name.__all__``} , {in REFGUIDE*}, and {missing from `others`}

Reply:
Rectified